lib/components/narrative/tabbed-itineraries.js

```diff
@@ -64,49 +66,57 @@ class TabbedItineraries extends Component {
         !useRealtime
       )
     return (
-      <div className='options itinerary tabbed-itineraries'>
-        <div className='tab-row'>
-          {itineraries.map((itinerary, index) => {
-            return (
-              <TabButton
-                currency={currency}
-                defaultFareKey={defaultFareKey}
-                index={index}
-                isActive={index === activeItinerary}
-                itinerary={itinerary}
-                key={index}
-                onClick={setActiveItinerary}
-              />
-            )
-          })}
-        </div>
+      <>
+        {error && itineraries && <div className='error-message'>
+          <div className='header'>
+            <i className='fa fa-exclamation-circle' /> Warning
```

Comment:
Use `<Icon type="..." withSpace />`, and move the "Warning" text inside the i18n message YML.

Reply:
Removed in ae51c07e1e6bbf86d5e1edd1c32408e0ef025295